service/src/main/kotlin/fi/espoo/evaka/shared/dev/DevApi.kt

```diff
@@ -554,6 +554,7 @@ RETURNING id
 
     @DeleteMapping("/message/delete-all")
     fun deleteMessages(db: Database) {
+        db.transaction { it.execute("DELETE FROM message_draft") }
```

Comment:
Doesn't really matter but the same transaction could be used for all these operations.